.github/workflows/reusable-golangci-lint.yml

```diff
@@ -22,13 +22,13 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Setup go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
           go-version: ${{ inputs.go-version }}
      
       - name: golangci-lint-${{inputs.directory}}
-        uses: golangci/golangci-lint-action@v3.2.0
+        uses: golangci/golangci-lint-action@v3.4.0
         with:
-          version: v1.46.2
+          version: v1.51
```

Comment:
as per the docs, it only supports specifying the minor version, it'll automatically pick the latest patch

Reply:
https://github.com/golangci/golangci-lint-action